v1.6.2/README.md

```diff
@@ -0,0 +1,23 @@
+# Canonical ROCKs for Longhorn v1.6.2
+
+Aim to be compatible with following upstream images
+[listed in the v1.6.2 release of Longhorn](https://github.com/longhorn/longhorn/releases/download/v1.6.2/longhorn-images.txt).
+
+* longhornio/longhorn-engine:v1.6.2
+* longhornio/longhorn-manager:v1.6.2
+* longhornio/longhorn-ui:v1.6.2
+* longhornio/longhorn-instance-manager:v1.6.2
+* longhornio/longhorn-share-manager:v1.6.2
+* longhornio/backing-image-manager:v1.6.2
+* longhornio/support-bundle-kit:v0.0.37
+
+The CSI components required by Longhorn can be found here: https://github.com/canonical/csi-rocks.
+
+The CSI rocks required for a Longhorn v1.6.2 deplyoment are:
+
+* ghcr.io/canonical/csi-attacher:v4.5.1
+* ghcr.io/canonical/csi-provisioner:v3.6.4
+* ghcr.io/canonical/csi-node-driver-registrar:v2.9.2
+* ghcr.io/canonical/csi-resizer:v1.10.1
+* ghcr.io/canonical/csi-snapshotter:v6.3.4
+* ghcr.io/canonical/livenessprobe:v2.12.0
```

Comment:
These images do not exist. First of all, we don't have v's in them. Secondly, we have to specify the ``-ckX`` suffix.

The list is:

```
* ghcr.io/canonical/csi-attacher:4.5.1-ck4
* ghcr.io/canonical/csi-provisioner:3.6.4-ck0
* ghcr.io/canonical/csi-node-driver-registrar:2.9.2-ck1
* ghcr.io/canonical/csi-resizer:1.10.1-ck6
* ghcr.io/canonical/csi-snapshotter:6.3.4-ck1
* ghcr.io/canonical/livenessprobe:2.12.0-ck6
```

Reply:
I know, but from my PoV we should have the `-ckN` thing in the first place (or if we do, we should have a plain tag which points to the right one)